src/data/community-events.json

```
@@ -240,6 +240,14 @@
     "location": "Rio de Janeiro, Brazil",
     "description": "3rd Edition of ETH Rio. ETH Rio 2024 will bring regulators, builders and businesses to discuss the future of the Brazilian Tokenized Economy based on EVM."
   },
+  {
+    "title": "ETH Beijing Hackathon",
+    "startDate": "2024-05-17",
+    "endDate": "2024-05-19",
+    "to": "https://www.ethbeijing.xyz/",
+    "location": "Beijing, China",
+    "description": "2nd Edition of ETH Beijing.The biggest Ethereum community event in Beijing, China. Hosted by PKU Blockchain and WTF Academy."
```

- The 'hosted by doesn't seem relevant' to users.
- Some copy clean up

```suggestion
    "description": "Second edition of ETH Beijing. The biggest Ethereum community event in Beijing."
```